honeybee_radiance_folder/folder.py

```diff
@@ -423,6 +423,30 @@ def aperture_files(self, black_out=False, rel_path=True):
         )
         return self._match_files(modifier_files, geometry_files)
 
+    def aperture_group_files_black(self, exclude=None, rel_path=True):
+        """Return list of files for black aperture groups.
+
+        Args:
+            exclude: Identifier of aperture group to exclude from the list of black
+                aperture groups files. This input can be either a single aperture group
+                identifier or a list of aperture group identifiers.
+            rel_path (str): Set rel_path to False for getting full path to files. By
+                default the path is relative to study folder root.
+        """
+        if exclude:
+            if not isinstance(exclude, (list, tuple)):
+                exclude = [exclude]
+            if len(exclude) == 1:
+                pattern = '(?!%s)^.*\..black.rad$' % exclude[0]
+            else:
+                pattern = '(?!%s)^.*\..black.rad$' % '|'.join(exclude)
+        else:
+            pattern = '^.*\..black.rad$'
```

Comment:
We should extract the files from the state file.  They may or may not follow this pattern for naming.